aws/resource_aws_config_remediation_configuration_test.go

```diff
@@ -0,0 +1,184 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func testAccConfigRemediationConfiguration_basic(t *testing.T) {
+	var rc configservice.RemediationConfiguration
+	resourceName := "aws_config_remediation_configuration.foo"
+	rInt := acctest.RandInt()
+	expectedName := fmt.Sprintf("tf-acc-test-%d", rInt)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckConfigRemediationConfigurationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfigRemediationConfigurationConfig_basic(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckConfigRemediationConfigurationExists("aws_config_remediation_configuration.foo", &rc),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "config_rule_name", expectedName),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "target_id", "SSM_DOCUMENT"),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "target_type", "AWS-PublishSNSNotification"),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "parameters.#", "2"),
```

Comment:
We can use `resourceName` here
```suggestion
					testAccCheckConfigRemediationConfigurationExists(resourceName, &rc),
					resource.TestCheckResourceAttr(resourceName, "config_rule_name", expectedName),
					resource.TestCheckResourceAttr(resourceName, "target_id", "SSM_DOCUMENT"),
					resource.TestCheckResourceAttr(resourceName, "target_type", "AWS-PublishSNSNotification"),
					resource.TestCheckResourceAttr(resourceName, "parameters.#", "2"),
```